src/Microsoft.Diagnostics.Monitoring.WebApi/Controllers/DiagController.cs

```diff
@@ -33,7 +33,7 @@ namespace Microsoft.Diagnostics.Monitoring.WebApi.Controllers
     [ProducesResponseType(typeof(void), StatusCodes.Status401Unauthorized)]
     public partial class DiagController : DiagnosticsControllerBase
     {
-        private const TraceProfile DefaultTraceProfiles = TraceProfile.Cpu | TraceProfile.Http | TraceProfile.Metrics;
+        private const TraceProfile DefaultTraceProfiles = TraceProfile.Cpu | TraceProfile.Http | TraceProfile.Metrics | TraceProfile.Gc;
```

Comment:
I have concerns about making this the default; how expensive/verbose is the data? Does it make sense to pair gc analysis together with cpu analysis?

Reply:
GC Collect Only traces are typically very lightweight. It only emits a handful of events per GC. Especially so when compared with CPU traces, where the runtime typically emit an event with stack per millisecond.